azurerm/internal/services/machinelearning/parse/inference_cluster.go

```diff
@@ -7,7 +7,7 @@ import (
 )
```

Comment:
How can I (possibly) resolve this golint error on line 5? - All other linting-related errors should have been addressed